README.md

```diff
@@ -111,6 +112,21 @@ Here is a [branch](https://github.com/transitive-bullshit/react-modern-library-b
 
 Here is a [branch](https://github.com/transitive-bullshit/react-modern-library-boilerplate/tree/feature/material-ui) which demonstrates how to create a module that makes use of a relatively complicated peer dependency, [material-ui](https://github.com/mui-org/material-ui). It shows the power of [rollup-plugin-peer-deps-external](https://www.npmjs.com/package/rollup-plugin-peer-deps-external) which makes it a breeze to create reusable modules that include complicated material-ui subcomponents without having them bundled as a part of your module.
 
+## Sourcemaps
```

Comment:
I'm fine with adding sourcemap generation by default, but I'd rather not add a whole section to the readme devoted to them.  imho, the walkthrough instructions are already verbose enough.